mongodbatlas/resource_mongodbatlas_advanced_cluster.go

```diff
@@ -345,6 +341,31 @@ func resourceMongoDBAtlasAdvancedCluster() *schema.Resource {
 	}
 }
 
+var tagsSchema = schema.Schema{
+	Type:     schema.TypeSet,
+	Optional: true,
+	Elem: &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"key": {
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
Currently defining key and value attributes as required based on my investigation in mms, but will confirm with upstream team.

Reply:
Confirmed both `key` and `value` attributes in tags are required (CLOUDP-199951).